quantifier/src/main/java/edu/illinois/cs/cogcomp/quant/lbj/QuantitiesClassifier$$1.java

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
We should think about this policy; I'll create an issue.  This module is fine for now, though.